distribution/ddf-common/src/main/resources/etc/certs/CertNew.cmd

```diff
@@ -1,12 +1,26 @@
-REM Usage: CertNew -cn some.computer.com
-REM    or  CertNew -dn "cn=some.computer.com,o=some company,ou=some department"
-REM The first usage creates a certificate with a subject/common name of some.computer.com.
-REM The second usage creates a certificate with the FQDN provided and extracts the common name
+REM Usage:
+REM   CertNew.sh [-cn <cn>|-dn <dn>] [-san <tag:name,tag:name,...>]
+REM
+REM where:
+REM <cn> represents a fully qualified common name (e.g. "<FQDN>", where <FQDN> could be something like cluster.yoyo.com)
+REM <dn> represents a distinguished name as a comma-delimited string (e.g. "c=US, st=California, o=Yoyodyne, l=San Narciso, cn=<FQDN>")
+REM <tag:name,tag:name,...> represents optional subject alternative names to be added to the generated certificate
+REM    (e.g. "DNS:<FQDN>,DNS:node1.<FQDN>,DNS:node2.<FQDN>"). The format for subject alternative names
+REM    is similar to the OpenSSL X509 configuration format. Supported tags are:
+REM      email - email subject
+REM      URI - uniformed resource identifier
+REM      RID - registered id
+REM      DNS - hostname
+REM      IP - ip address (V4 or V6)
+REM      dirName - directory name
 REM
 REM Create new certificate and certificate chain signed by Demo Certificate Authority.
 REM The new certificate chain and private key are installed in the keystore.
 REM The alias will be the same as the common name.
 REM The localhost key will be deleted from the keystore.
+REM If no arguments specified on the command line, `hostname -f` is used as the
```

Comment:
Removed it from the Windows command as it doesn't do that. Updated the doc for the Linux command to document it. BTW, this is not a new feature added by this PR 😉